### PR TITLE
Remove unused pages

### PR DESCRIPTION
**Pull request by lifehackerhansol**
**Description**

With many different updates and many new ways to hack the console, there exists many pages that do not have much use other than "legacy!"

This has caused a number of issues that are sort of related to the guide:
- Old video guides, or old forum posts, directing to the outdated page in question
- helpees at Nintendo Homebrew :briccaww:
- Translations not being kept up to date, and the translation team's strict requirements on the whole guide being translated, leading to many languages not being activated or many broken translations
- the usual "old stuff don't work"

With that in mind, this pull request aims to remove all these pages. As well, the 404 page will be updated to redirect people who may have mistakenly entered the page to the ~~Get Started~~ site navigation page.

The following list will be updated accordingly to the progress of this pull request.

**Pages that were already stubbed out removed in this PR:**
- DSiDumper
- Browserhax 2020
- HBL (steelhax)
- B9S (HBL-USM)

**Pages requiring a service that has been discontinued removed in this PR:**
- B9S (safecerthax)
- B9S (ssloth-browser)

**Pages requiring outdated userland exploits other than soundhax removed in this PR:**
- HBL (alternatives)

**Pages that don't even have a meaning anymore removed in this PR:**
- Cartridge Update

**Pages that have been superseded by other methods removed in this PR:**
- B9S (mset)
- B9S (2xrsa)
- B9S (frogtool)
- HBL (PicHaxx)

**Pages with an exploit that is permanently patched removed in this PR:**
- B9S (Browser)

**Pages that keep being added by Crowdin removed in this PR:**
- B9S (frogminer)
- B9S (HBL)

This PR also rebases the `assets/js` folder with Minimal Mistakes upstream as `main.min.js` refused to build otherwise.

### CODE REVIEW

**Comment by lifehackerhansol** [comment]
All pages described have been removed in this PR. If any else should be removed feel free to directly commit to the PR or submit a review.

File: _pages/en_US/404.txt [hunk 1]
```diff
@@ -5,5 +5,5 @@ excerpt: "Page not found. Your pixels are in another canvas." #
 sitemap: false
 ---
 
-Sorry, but the page you were trying to view does not exist.
+Sorry, but the page you were trying to view does not exist. This may be because the page was removed due to being outdated.
```
Comment (lilyuwuu) [inline _pages/en_US/404.txt:8]: A bit oddly worded, and would like to link people to get started or the homepage.

A problem with just wording it as a maybe is that discord still drops closing brackets unless you append # or .html, meaning we could have any current link also 404, which would confuse people into thinking whatever current method is outdated.

**Review by lilyuwuu — COMMENTED** [review]
Removed pages lgtm

**Comment by lifehackerhansol** [comment]
404 now redirects to site navigation page. 

Ideally in this case, the site navigation page should be better organized. But at least this way people will be able to see a list of pages and get the gist.

**Comment by radostin04** [comment]
Shouldn't we keep  these pages around somewhere? There's still cases where some, esepcially USM, are useful. Maybe clone the current version of the guide somewhere else? Not sure if a second quantumcat is a good idea thon

**Comment by lifehackerhansol** [comment]
It would be more ideal to keep the pages in a separate site, at least with this current setup. If we end up moving to another engine or theme it can be added back to a new legacy section, I guess?

**Comment by lifehackerhansol** [comment]
With the new 11.15 update, seedminer has been brought back from the grave. ~~Please fkin stop Nintendo~~

**Comment by lifehackerhansol** [comment]
If we are to merge this, then the 1115 branch should be merged first, so I can update this PR to match.

**Comment by lifehackerhansol** [comment]
Removed some more pages, just the actually-unused Seedminer pages.

**Comment by lifehackerhansol** [comment]
Oh yeah I forgot to mention, the browser method is also deleted here.

**Comment by MechanicalDragon0687** [comment]
no plans to update the sidebar path that references the navigation yml? 
https://github.com/hacks-guide/Guide_3DS/blob/master/assets/js/_main.js#L135
and
https://github.com/hacks-guide/Guide_3DS/blob/master/assets/js/_main.js#L184

**Comment by lifehackerhansol** [comment]
You know, I forgot those existed. I'll do that in a sec

**Comment by lifehackerhansol** [comment]
Updated main.js. Didn't update min.js, will do that soon:tm:

**Comment by lifehackerhansol** [comment]
Completed. Re-requesting review.

**Comment by lifehackerhansol** [comment]
Squashed the commits down to 3, and rebased with latest guide upstream.

File: assets/js/_main.js [hunk 2]
```diff
@@ -132,26 +132,19 @@ $(document).ready(function(){
   // the value is the key to the displayed route in the device_common/old/new variable below
   // 
   var methods = {
```
Comment (MechanicalDragon0687) [inline assets/js/_main.js:134]: You have to match the methods to the later paths

Comment (lifehackerhansol) [reply assets/js/_main.js:134]: Resolved. (I think.)

**Comment by lifehackerhansol** [comment]
Per suggestion of dealing with the javascript stuff later I temporarily rolled back to MM 4.6.0 and built main.min.js.

**Comment by lifehackerhansol** [comment]
I've also fixed package.json to use the existing js modules.

**Comment by lifehackerhansol** [comment]
Engine is fixed (mostly) and PR is back open.

**Pull request merged**